src/server/req_delete.c

```diff
@@ -487,14 +493,7 @@ req_deletejob(struct batch_request *preq)
 		req_reject(PBSE_IVALREQ, 0, preq);
 		return;
 	}
-	/*
-	 * If the request if to also purge the history of the sub job then log a message since
-	 * server does not store the history of subjobs
-	 */
-	if (delhist)
-		log_event(PBSEVENT_DEBUG, PBS_EVENTCLASS_JOB,
-			LOG_INFO, jid,
-			msg_nosave_histarrayjob);
+
```

Comment:
I think the following log message is still required unless we announce it in EDD.
"History of individual array subjobs not saved"

Reply:
I have updated the EDD with this change https://pbspro.atlassian.net/wiki/spaces/PD/pages/161873965/External+interface+design+for+subjobs+surviving+server+restarts.

Reply:
Please also make sure that there is a test case(if not present already) to test the behavior

Reply:
extend test_job_array_comment ptl test to verify running subjob job comment in rc 29 https://github.com/PBSPro/pbspro/pull/590/commits/a443458a722e843403083a3ac3efdb29bf80e8da